### ISSUE TITLE
Not compatible with Gradle 5.1

### ISSUE THREAD

**Issue by jcayzac** [evant/gradle-retrolambda#274]
The plugin fails when it's invoked by Gradle 5.1:

```
java.lang.NoSuchMethodError: org.gradle.api.tasks.compile.CompileOptions.getBootClasspath()Ljava/lang/String;
        at me.tatarka.RetrolambdaTransform.getClasspath(RetrolambdaTransform.groovy:126)
```

**Comment by evant** [comment]
It has been renamed to `getBootstrapClasspath()`

**Comment by naixx** [comment]
Any chance that will be fixed?
Or any direction to investigate? I tried to change the property at RetrolambdaTransform.groovy#L126 but no luck

**Comment by evant** [comment]
@naixx pull requests are welcome, changing that line you gave _should_ be the solution. In what way is it not working?

**Comment by jcayzac** [comment]
Not sure, but this might relate to https://docs.gradle.org/current/userguide/task_configuration_avoidance.html

**Comment by daniebruwer** [comment]
I'm also having the same issue. Is the solution to fork? Is there a newer version than 3.7.0 that avoids this?
classpath 'com.android.tools.build:gradle:3.5.0-alpha07'
distributionUrl=https\://services.gradle.org/distributions/gradle-5.3-all.zip

**Comment by evant** [comment]
Released version 3.7.1 which should be compatible with gradle 5+

**Comment by jcayzac** [comment]
Thanks!
